dev/com.ibm.websphere.appserver.thirdparty.eclipselink.3.0/bnd.bnd

```diff
@@ -98,7 +99,7 @@ DynamicImport-Package: \
  javax.xml.bind.helpers, \
  javax.xml.bind.util
 
-Private-Package: com.ibm.ws.eclipselink.osgi
+Private-Package: io.openliberty.persistence.internal.helper
```

Comment:
What do you want to do with the `com.ibm.ws.i18n.conversion.tools` package that is in this project but appears to be excluded from the bundle?

Reply:
Keep them there, they're tools used to generate nlsprops files from Eclipselink resource bundles.  They're not shipped with the product.